opentech/static_src/src/app/src/components/NotesPanelItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+export default class NotesPanelItem extends React.Component {
+    static propTypes = {
+        note: PropTypes.shape({
```

Comment:
Maybe just accept the `user` `timestamp` `message` as props. Unsure about this one.